examples/onnxruntime/training/image-classification/run_image_classification.py

```diff
@@ -109,6 +110,14 @@ class DataTrainingArguments:
             )
         },
     )
+    image_column_name: str = field(
+        default="image",
+        metadata={"help": "The name of the dataset column containing the image data. Defaults to 'image'."},
+    )
+    label_column_name: str = field(
+        default="label",
+        metadata={"help": "The name of the dataset column containing the labels. Defaults to 'label'."},
+    )
```

Comment:
I think you may need to add `--label_column_name labels` to the command in the [README](https://github.com/huggingface/optimum/tree/main/examples/onnxruntime/training/image-classification) due to this change, can you try it out please?